src/main/java/org/ays/common/util/exception/handler/GlobalExceptionHandler.java

```diff
@@ -160,6 +160,7 @@ AysErrorResponse handleAccessDeniedError(final AccessDeniedException exception)
 
         return AysErrorResponse.builder()
                 .header(AysErrorResponse.Header.AUTH_ERROR.getName())
+                .message(exception.getMessage())
```

Comment:
@suhakopan burada Access Denied için message dönmektense yeni bir tane `AysForbiddenException` açıp asıl sınıfı oradan extend edelim. Bu sayede daha düzgün olur. Bir öncekinde hata aslında buradaydı.

Reply:
Bunu kaldırıp problemin asıl yeri olan AysUserNotSuperAdminException için exception handler yazsak doğru olmaz mı?

Reply:
@suhakopan her exception'ı handle etmeye kalkarsak muhtemelen yönetemeyiz, bu yüzden base exception'ları belirleyip sadece extend etmek yeterli olur, çünkü ilerleyen dönemde başka exception'lar da gelirse onları da eklememiz gerekecek